src/helpers.h

```diff
@@ -6,6 +6,7 @@
 #include "liteclient.h"
 
 void generate_correlation_id(Uptane::Target& t);
+std::string generate_correlation_id(TufTarget& t);
```

Comment:
Another question/topic, not related to this PR:
1. Can we make cor ID optional paramater and auto-generate inside `PullAndInstall`?
2. How a user can "know" what is cor ID if he/she/it calls pull and install separately?

Reply:
> * Can we make cor ID optional paramater and auto-generate inside `PullAndInstall`?
Yes, that makes sense. Right now an empty `correlation_id` leads to it being auto-generated inside the `Installer` constructor. I think it is worth to set some default values in `PullAndInstall` in the current PR, I'll do it.

> * How a user can "know" what is cor ID if he/she/it calls pull and install separately?
Currently I believe there is no way, the user would have to generate and pass the same ID to both calls. Install finalization falls in a similar situation but, in that case, there is an entry in `installed_versions` table where the cor ID is fetched from.